src/index.html

```diff
@@ -9,6 +9,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#image">
+        <img
+        class="image"
+        src="images\moyo_1.png"
+        alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item li_is_active">
+            <a href="#Apple" class="nav-link is-active">Apple</a>
+          </li>
+          <li class="nav-item">
```

Comment:
```suggestion
          </li>
          
          <li class="nav-item">
```

Same as above and fix it everywhere